### PR TITLE
[ Ent-Corda ] Node registration helm chart

### PR DESCRIPTION
**Pull request by suvajit-sarkar**
**Changelog**
- Add Node registration

 

**to be reviewed by**
@deepakkumardbd @jagpreetsinghsasan 

 

**Linked issue**
#901

### CODE REVIEW

File: platforms/corda-ent/charts/node-initial-registration/files/node-initial-registration.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    HOW_LONG={{ .Values.sleepTimeAfterError }}
+    echo
+    echo "Node initial registration failed - exit code: ${EXIT_CODE} (error)"
+    echo
+    echo "Going to sleep for requested ${HOW_LONG} seconds to let you login and investigate."
+    echo
+    pwd
+    ls -al 
+else
+    HOW_LONG={{ .Values.sleepTime }}
```
Comment (deepakkumardbd) [inline platforms/corda-ent/charts/node-initial-registration/files/node-initial-registration.sh:67]: we can remove this from values as sleepTime is always 0 and in that case container is not sleeping.

Comment (sownak) [reply platforms/corda-ent/charts/node-initial-registration/files/node-initial-registration.sh:67]: Why is sleepTime always 0?

File: platforms/corda-ent/charts/node-initial-registration/values.yaml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  # Provide the vault orginsation node folder name where the certificates are stored
+  # Eg. nodePath: carrier
+  nodePath:
+
+# Provide volume related specifications
+volume:
+ # Ex baseDir: /opt/corda
+  baseDir:
+# where is node jar is stored
+jarPath: bin
```
Comment (sownak) [inline platforms/corda-ent/charts/node-initial-registration/values.yaml:66]: these also should be in settings section

Comment (suvajit-sarkar) [reply platforms/corda-ent/charts/node-initial-registration/values.yaml:66]: ok working on this

File: platforms/corda-ent/charts/node-initial-registration/files/node-initial-registration.sh (hunk 1 above)
Comment (suvajit-sarkar) [reply platforms/corda-ent/charts/node-initial-registration/files/node-initial-registration.sh:67]: the valuefile had default value 0, have changed it

File: platforms/corda-ent/charts/node-initial-registration/values.yaml (hunk 2 above)
Comment (suvajit-sarkar) [reply platforms/corda-ent/charts/node-initial-registration/values.yaml]: done, moved it to settings section

**Pull request merged**